harc_game_web/apps/tasks/models.py

```diff
@@ -210,3 +223,9 @@ def update_profile_signal(sender, instance, created, **kwargs):
         )
     instance.taskapproval.save()
 
+def close_task_approvals():
+    """
+    Called by cron at 00:00 on Saturday to close the task approvals
+    """
+    # Pick tasks, which are not closed and completed in the past and close them
+    TaskApproval.objects.filter(is_closed=False).exclude(documented_task__date_completed__date=timezone.now()).update(is_closed=True)
```

Comment:
`documented_task__date_completed__date=timezone.now()`? tu nie powinno być jakiegoś `gte` czy `lte`?

Reply:
W sumie to mogło by być. W normalnym przypadku wyraża się to po północy z piątku na sobotę i jest zabezpieczenie na przypadek, ża akurat ktoś zrobił dokumentację i review w czasie, gdy cron procesował bazę. Ale mogę dodać na wypadek jakiś sytuacji, że ręcznie coś jest dodawane w przyszłości (ale po co miało by być?).

Reply:
Nie powinno być też __date, bo to zaokrągla do całego dnia, a gdyby schedule był o innej godzinie niż północ, to...